app.py

```diff
@@ -61,8 +62,8 @@ def __init__(
         self.checked_repos: Set[str] = set()
 
         deployment_name = os.environ["THOTH_DEPLOYMENT_NAME"]
-        self.kebechet_path = Path(f"{deployment_name}/{subdir}/thoth-sli-metrics/kebechet-update-manager/")
-        self.mi_path = Path(f"{deployment_name}/mi/{subdir}")
+        self.kebechet_path = str(Path(deployment_name).joinpath(subdir).joinpath(KEBECHET_KNOWLEDGE_PATH))
```

Comment:
```
deployment_path = Path(deployment_name)
mi_path = deployment_path.joinpath("mi")

if subdir:
      self.kebechet_path = deployment_path.joinpath(subdir).joinpath(KEBECHET_KNOWLEDGE_PATH)).as_posix()
      self.mi_path = mi_path.joinpath(subdir).as_posix()
else:
     self.kebechet_path = deployment_path.joinpath(KEBECHET_KNOWLEDGE_PATH)).as_posix()
     self.mi_path = mi_path.as_posix()
```


could you move `deployment_name = os.environ["THOTH_DEPLOYMENT_NAME"]` to start of the file as other env variables and make it with capital letter please? It is more readable in this way

Reply:
done :) no need for if esle branching, if the subdir is "" it is alright, and if it is None, then the user misuses the class (same reason why the if guard in `main` was removed)